### PR TITLE
Separate config from images

### PR DESCRIPTION
**Pull request by heronimus**
Issue:  https://github.com/asiantbd/crypto-discord-bot/issues/5

Changelog:
- Dockerfile: remove unused config file on build
- Viper Config: add alternate dir ./config/ 
- Unit Test: add alternate dir test scenario
- Deployments: remove production config from images, create config-volume
- Config: update example file

### CODE REVIEW

File: main_test.go [hunk 1]
```diff
@@ -9,9 +9,10 @@ import (
 )
 
 func setViperTest() error {
```
Comment (joshuagibeonm) [inline main_test.go:11]: Mending dihapus skalian aja, yg di main_test juga pake setviper yg di main.go

Comment (heronimus) [reply main_test.go:11]: oiya gaada yg di test juga disini wkwk

**Review by joshuagibeonm — APPROVED** [review]
LGTM

File: main_test.go (hunk 1 above)
Comment (joshuagibeonm) [reply main_test.go]: lull

**Pull request merged**